### PR TITLE
Modular Computer Enhancement

### PR DESCRIPTION
**Pull request by Geevies**
* Modular computers have gotten a major clean-up. Many programs and interactions should look prettier and feel smoother now.
* Click dragging most modular computers now open their interface, much like with PDAs.
* The IRC program now outputs messages to your chat log, much like PDA messaging does. For smaller devices, it must be on your person, whilst larger ones have a radius to it. The IRC program must be open, it doesn't have to be active, for it to work.
* The size of the antag hacked camera program has been brought way down, to make it worth getting, if you wish to do so.
* Added some new names to the modular computer arcade game.
* Suit sensors can now be run on most regular modular computers, no big change in that department.
* Computers now ping when they finish downloading programs, letting you know without needing to check them.

### CODE REVIEW

**Comment by Geevies** [comment]
!review

**Comment by Arrow768** [comment]
Want to take a look at it before it's merged.
All the u separated style changes make reviewing that pr more difficult

**Comment by Arrow768** [comment]
And I fat fingered the "close and comment" button

**Comment by Geevies** [comment]
> 
> 
> And I fat fingered the "close and comment" button

the merge button's just a bit to the left ;)

File: code/modules/modular_computers/file_system/programs/command/comm.dm [hunk 1]
```diff
@@ -305,10 +311,10 @@ var/last_message_id = 0
 	for (var/datum/comm_message_listener/l in comm_message_listeners)
 		l.Add(message)
 
-	for (var/obj/item/modular_computer/computer in get_listeners_by_type("modular_computers", /obj/item/modular_computer))
-		if(computer && computer.working && !!computer.nano_printer)
+	for (var/obj/item/modular_computer/computer in get_listeners_by_type(LISTENER_MODULAR_COMPUTER, /obj/item/modular_computer))
+		if(computer?.working && !!computer.nano_printer && computer.hard_drive?.stored_files.len)
```
Comment (Arrow768) [inline code/modules/modular_computers/file_system/programs/command/comm.dm:315]: What is the purpose of the added check here?

Comment (Geevies) [reply code/modules/modular_computers/file_system/programs/command/comm.dm:315]: I tested the code locally on Runtime, and it runtimed at that exact spot. The check stops the runtime from happening

**Pull request merged**